test/helpers/kubectl.go

```diff
@@ -1365,9 +1365,16 @@ func (kub *Kubectl) CiliumReport(namespace string, commands ...string) {
 		kub.logger.WithError(err).Error("cannot retrieve cilium pods on ReportDump")
 		return
 	}
+
 	res := kub.Exec(fmt.Sprintf("%s get pods -o wide --all-namespaces", KubectlCmd))
 	ginkgoext.GinkgoPrint(res.GetDebugMessage())
 
+	res = kub.Exec(fmt.Sprintf("%s get cep -o wide --all-namespaces", KubectlCmd))
```

Comment:
There were a reason for not included that on the cilium-report that makes the reading the xml output too big and getting context was super difficult. 

Also, more or less, all the test will print endpoint/policy if is needed due is passed on the arguments: 
https://github.com/cilium/cilium/blob/8ffa351f7cb88f8b2bf57be5c6951b8bca5c08ba/test/k8sT/Services.go#L61-L63

Reply:
What XML output? Can you give an example?

Reply:
Sorry, the XML output is the junit format. What Jenkins used. 

For example, you are duplicating here the endpoints (CEP may not be up to date) and the CNP is not needed at all. (See the standar output)

https://jenkins.cilium.io/job/Ginkgo-CI-Tests-Pipeline/5368/testReport/junit/k8s-1/8/K8sChaosTest_Restart_with_long_lived_connections_L3_L4_policies_still_work_while_Cilium_is_restarted/

Reply:
I don't understand the "too long" argument. Is this technical? The overview is absolutely nice and it doesn't matter if it requires some scrolling. Downloading zipfiles and digging in there is very time-consuming. I will split this into a separate PR so we can discuss in details there.